src/App.tsx

```diff
@@ -40,7 +40,7 @@ function App() {
       setRows(
         getPeopleQuery.data
           .map((person, index) => ({ id: index, ...person })) // Add a unique ID to each row
-          .sort((a, b) => a.name.localeCompare(b.name)), // Sort by name
+          .sort((a, b) => a.name.localeCompare(a.name)), // Sort by name
```

Comment:
**Suggestion:** Correct the sorting logic in the setRows function. It currently compares 'a.name' with itself, which is likely a mistake. It should compare 'a.name' with 'b.name'.
```suggestion
          setRows(
            getPeopleQuery.data
              .map((person, index) => ({ id: index, ...person })) // Add a unique ID to each row
              .sort((a, b) => a.name.localeCompare(b.name)), // Sort by name
          )
```